examples/image_classifier/streaming_video_client_side_batching/README.md

```diff
@@ -0,0 +1,80 @@
+# Streaming Video Inference with client-side batching
+
+To use TorchServe's dynamic batching feature, please refer to [resnet_152_batch](https://github.com/pytorch/serve/tree/examples/streaming_video/examples/image_classifier/resnet_152_batch)
+
+For some near real-time streaming video applications, there might be a need for batching on the application side.
```

Comment:
Please Describe when that need applies

Reply:
> there might be a need for batching on the application side
I suggest changing the "might" to an example/ use-case where it works.

Reply:
Changed the example to show both TS batching and client side batching